tests/bin/check/run.t

```diff
@@ -0,0 +1,194 @@
+A necessary condition for dune-release to work is that it's able to infer the main package
+
+    $ touch first_pkg.opam
+    $ touch other_pkg.opam
+
+    $ dune-release check --working-tree
+    [FAIL] cannot determine distribution name automatically: add (name <name>) to dune-project
+    [1]
+
+Let's clean up to not interfere with the following tests
+
+    $ rm first_pkg.opam other_pkg.opam
+
+If the condition described above is fulfilled, there are 4 checks to be performed
+
+    $ cat > my_pkg.opam <<EOF \
+    > opam-version: "2.0"\
+    > dev-repo: "git+https://github.com/fu/fa.git"\
+    > description : "Some description"\
+    > maintainer : "me"\
+    > authors : "also me"\
+    > homepage : "https://github.com/fu/fa"\
+    > bug-reports : "https://github.com/fu/fa/issues"\
+    > synopsis : "Dope project"\
+    > EOF
+    $ echo "(lang dune 2.7)" > dune-project
+
+    $ dune-release check --working-tree | ./make_lint_directory_deterministic.exe | ./make_test_directory_deterministic.exe
```

Comment:
Might be good to merge those two binaries into a single one, what do you think?

Reply:
We could even move it upwards in `tests/bin/helpers/sanitize_output.ml` so it can be enriched and reused accross our different cram tests!

What do you think about that?

Reply:
I've done both things. Thanks for the help!